test/test_quick_setup.py

```diff
@@ -23,3 +25,14 @@ def test_exp_quick_setup_freqs() -> None:
     carrier_freq = gate.comps["d1"]["carrier"].params["freq"].get_value()
     offset = gate.comps["d1"]["gaussian"].params["freq_offset"].get_value()
     assert qubit_freq == carrier_freq + offset
+
+@pytest.mark.integration
+def test_generator() -> None:
+    gen_signal = pmap.generator.generate_signals(pmap.instructions["rx90p[0]"])
+    with open("test/quick_data.pickle", "rb") as quickfile:
+        test_data = pickle.load(quickfile)
+    np.testing.assert_allclose(gen_signal["d1"]["ts"], test_data["d1"]["ts"])
+    np.testing.assert_allclose(
```

Comment:
One possible standard we might want to enforce is that original test writers should provide explicit `atol` or `rtol` values for tests that use `assert_allclose` type functionality. These tolerances should be the maximum acceptable worst case figures. This ensures that we don't tamper with tests later on when introducing some new feature or fixing a bug, with the assumption that the original test writer already accounted for the worst case figures and a failing test indicates something other than precision issues.